pitch.md

```diff
@@ -23,38 +37,125 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
+However, as mentioned before, this is designed for linguists.
+Qtree's documentation states that
+> The front end of _qtree_ reads a tree description written in the familiar (to linguists) bracket notation. 
+
+However, I found this bracket notation confusing.
+Furthermore, this package requires a lot of work-around to get it to draw tableau trees.
+[Here](https://github.com/cpence/latex-tableaux) is a page that explains how to wrangle qtree to make it work for tableau trees.
+Tree packages generally expect each node to be a single line, which is not the case for tableau trees.
+It would also be much nicer if users don't have to exit and enter math mode in order to typeset the tree.
 
 ## Suitability
 _What percentage of your time do you think will be spent directly engaging in
 the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+If I decide to leverage an existing tree typesetting tool, I estimate that 70% of my time would be spent engaging in the language aspect.
+I would need to learn how to interact with the tool, but once I figure that out I can focus on designing the interface to the tool.
+
+If I decide to write the program myself, as I thought I would in the project-ideas assignment, I would need to spend much more time on semantics.
+
+Initially, I thought that the ideal syntax was very obvious and straightforward.
+But from Anthony's critiques, I think there are still more design decisions to consider.
+Such as designating a character to invoke logic symbols.
+
+Graphviz is an example for an alternative syntax for drawing graphs, and thus trees.
+So there may be a better syntax for trees that I have not considered.
 
 ## Scope
 _How big an idea is this? How ambitious is this project?_
 
+The project has a very specific purpose, for a very specific user base.
+So I think the project is has a narrow enough scope.
+I don't think it's very ambitious.
 
 ## Challenges and opportunities
 _Why is this a good project? What are some challenges you expect to face? How
 might you overcome them? What are some warning signs that the project has gotten
 off track, and how will you get the project back on track if needed?_
+
+This is a good project because I have my own motivation for completing it.
+The project probably won't be completed in time for me to use it in class, but I think having to use the tools I currently have in class will keep me motivated.
+
+I expect some challenge in interfacing with a tree typesetting package, as I'm already having some trouble with them right now.
+Using a package as a host language is probably more challenging than using it to draw a specific tree.
+I hope to avoid this as much as possible by researching a lot of tools before settling on a host language.
```

Comment:
What do you think would be the ideal host language? What should be easy / difficult in this host language?